packages/cozy-procedures/src/containers/PersonalDataForm.jsx

```diff
@@ -4,7 +4,7 @@ import PersonalDataForm from '../components/PersonalDataForm'
 import { getSlice, update as updateFormData } from '../redux/personalDataSlice'
 
 const mapStateToProps = state => ({
-  formData: getSlice(state)
+  formData: getSlice(state).data
```

Comment:
I think it would be better to have a selector that specifically gets the `data` of the slice.